Say Their Names/Source/View/Home/Person/PersonPhotoCell.swift

```diff
@@ -8,19 +8,57 @@
 
 import UIKit
 
-class PersonPhotoCell: UICollectionViewCell {
-
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
+final class PersonPhotoCell: UICollectionViewCell {
+    
+    private let personImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        personImageView.image = nil
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setup() {
+        addSubview(personImageView)
+                
+        let constraints = [
+            personImageView.topAnchor.constraint(equalTo: topAnchor),
+            personImageView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            personImageView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            personImageView.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ]
+        
+        NSLayoutConstraint.activate(constraints)
```

Comment:
No need to declare single-use constants here 

```
NSLayoutConstraint.activate([
            personImageView.topAnchor.constraint(equalTo: topAnchor),
            personImageView.leadingAnchor.constraint(equalTo: leadingAnchor),
            personImageView.trailingAnchor.constraint(equalTo: trailingAnchor),
            personImageView.bottomAnchor.constraint(equalTo: bottomAnchor)
        ])
```

Reply:
it helps with type checking time, especially if we add type annotation to the `constraints` variable. not a big deal with 4 constraints, but if it's over 10, usually makes a difference